mk/spksrc.service.installer.dsm7

```diff
@@ -58,7 +58,12 @@ fi
 
 # Reload wizard variables stored by postinst
 if [ -r "${INST_VARIABLES}" ]; then
-    . "${INST_VARIABLES}"
+    # we cannot source the file to reload the variables, when values have special characters like <, >, ...
```

Comment:
Just an idea: avoid copy-paste and collect command source code in a "generic" script - then sourced from specific DSM implementation, to reuse functions and reduce such maintenance effort

Reply:
This is not so easy, as the service-installer files are copied as installer into the staging script folder.
To realize this idea, we would need to call functions for such duplicated code and put those into an additional script file to the package and soure this file into the service-installer as we already do for service-setup.

But I will give a try.

Reply:
done.

unfortunately the functions are not usable within `service-setup.sh` and `upgrade_uifile.sh`